corehq/apps/domain/views/base.py

```diff
@@ -66,6 +73,36 @@ def select(request, do_not_redirect=False, next_view=None):
         return render(request, domain_select_template, additional_context)
 
 
+Link = namedtuple('Link', ('name', 'url'))
+
+
+@quickcache(['couch_user.username'])
+def get_domain_dropdown_links(couch_user, view_name="domain_homepage"):
+    domains = Domain.active_for_user(couch_user)
+    domain_links = [Link(
+        name=domain_obj.display_name(),
+        url=reverse(view_name, args=[domain_obj.name]),
+    ) for domain_obj in domains]
+
+    linked_domain_links = []
+    domain_names = {d.name for d in domain_links}
+    for domain_obj in domain_links:
+        domain = domain_obj.name
+        if toggles.ENTERPRISE_LINKED_DOMAINS.enabled(domain):
+            links = get_linked_domains(domain)
+            links = [link for link in links if link.linked_domain not in domain_names]
+            linked_domains = [Domain.get_by_name(link.linked_domain) for link in links]
+            linked_domain_links.extend([Link(
```

Comment:
i dont have a better word to suggest, but the use of link in the sense of "domain linking" and the sense of "html link" is mildly confusing here, especially with names like `linked_domain_links` that encompass both

Reply:
Agreed this is confusing. Added fe0cc2197b1 to favor "enterprise" over "linked" when describing domains. Enterprise is also overloaded but then at least the same code isn't using "linked" and "link" in very different ways. Open to further feedback.